src/test/util/blockfilter.cpp

```diff
@@ -28,4 +28,3 @@ bool ComputeFilter(BlockFilterType filter_type, const CBlockIndex* block_index,
     filter = BlockFilter(filter_type, block, block_undo);
     return true;
 }
-
```

Comment:
Why remove the trailing newline?

Reply:
We don't normally have trailing newlines in files.